src/views/App/App.jsx

```diff
@@ -17,7 +18,13 @@ export default (props) => {
       <header className="App-header">
         {appName}
       </header>
-      <RegisterPage />
+      <BrowserRouter>
+        <Switch>
+          <Route path="/" exact component={RegisterPage}/>
+          <Route path="/dashboard" exact/>
+          <Route path="/" render={() => <div>404</div> }/>
```

Comment:
doesn't this contradict the first route?

maybe something like this?
```suggestion
          <Route path="/error" render={() => <div>404</div> }/>
```

Reply:
It doesn't contradict it (I just tested again to be sure), do you still want me to change it to "/error"?, I put it as "/" so every path that is not registered to the one we have will reroute to page 404

Reply:
Ok i understand now. No let's keep it as it is